gettsim/policy_environment.py

```diff
@@ -295,9 +295,9 @@ def load_reforms_for_date(date):
         functions["kindergeld_anspruch"] = kindergeld_anspruch_nach_lohn
 
     if year > 2011:
-        functions["sonderausgaben"] = eink_st_sonderausgaben_ab_2012
+        functions["eink_st_sonderausgaben_tu"] = eink_st_sonderausgaben_ab_2012_tu
```

Comment:
For the other variables, the order is different -> `eink_st_sonderausgaben_tu_ab_2012`

Can you please make this change and then check if the order is consistent for all other variables in this file?